ui/imports/shared/panels/StatusAssetSelector.qml

```diff
@@ -165,5 +182,27 @@ Item {
                 }
             }
         }
+
+        Component.onCompleted: {
+            control.currentIndex = -1
+            control.popup.contentItem.header = searchBox
```

Comment:
Why not just declare them, instead of assigning them via imperative code?

Reply:
it doesnt work, tried that first instead of doing it here!